add powershell script (#486)
OctopusDeploy/docs: docs/deployment-examples/azure-deployments/deploying-to-azure-via-a-firewall/index.md

All the Azure steps in Octopus are executed from the VM where the Octopus Server is running. So to able to successfully deploy to the Microsoft cloud, you need to make sure your Octopus Server can reach it through the network.

> To check if you are able to reach Microsoft cloud through your network, try running this script on the same machine and using an account with the same permissions as your Octopus Server.

Reviewer: Suggestion to remove a few extra words:
```suggestion
To check you can reach Microsoft cloud through your network, run this script on the same machine using an account with the same permissions as your Octopus Server.
```

:::info

> You might need to install Azure Powershell before running this script. More information on how to do this can be found at https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-2.5.0

Reviewer: Updated the link formatting:
```suggestion
You might need to install Azure Powershell before running this script. For information on how to do, see [Install the Azure PowerShell module](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-2.5.0).
```

To check you can reach Microsoft cloud through your network, run this script on the same machine using an account with the same permissions as your Octopus Server.

:::info

You might need to install Azure Powershell before running this script. For information, see [Install the Azure PowerShell module](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-2.5.0).

:::

```
$ErrorActionPreference = "Stop"

$OctopusAzureADTenantId = #Enter TenantId here
$OctopusAzureSubscriptionId = #Enter SubscriptionId here
$OctopusAzureADClientId = #Enter ClientId here
$OctopusAzureADPassword = #Enter Secret here
$OctopusAzureEnvironment = "AzureCloud"

$securePassword = ConvertTo-SecureString $OctopusAzureADPassword -AsPlainText -Force
$creds = New-Object System.Management.Automation.PSCredential ($OctopusAzureADClientId, $securePassword)
$AzureEnvironment = Get-AzEnvironment -Name $OctopusAzureEnvironment

Connect-AzAccount -Credential $creds -TenantId $OctopusAzureADTenantId -SubscriptionId $OctopusAzureSubscriptionId -Environment $AzureEnvironment -ServicePrincipal

Get-AzResourceGroup
```

> You should see output showing all the Azure Resource Groups you have access to:

Reviewer: ```suggestion
If everything is working as expected, you will see output showing all the Azure Resource Groups you have access to:
```

To check if you are able to reach Microsoft cloud through your network, try running this script on the same machine and using an account with the same permissions as your Octopus Server.

:::info

You might need to install Azure Powershell before running this script. For information, see [Install the Azure PowerShell module](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-2.5.0).

:::

```
$ErrorActionPreference = "Stop"

$OctopusAzureADTenantId = #Enter TenantId here
$OctopusAzureSubscriptionId = #Enter SubscriptionId here
$OctopusAzureADClientId = #Enter ClientId here
$OctopusAzureADPassword = #Enter Secret here
$OctopusAzureEnvironment = "AzureCloud"

$securePassword = ConvertTo-SecureString $OctopusAzureADPassword -AsPlainText -Force
$creds = New-Object System.Management.Automation.PSCredential ($OctopusAzureADClientId, $securePassword)
$AzureEnvironment = Get-AzEnvironment -Name $OctopusAzureEnvironment

Connect-AzAccount -Credential $creds -TenantId $OctopusAzureADTenantId -SubscriptionId $OctopusAzureSubscriptionId -Environment $AzureEnvironment -ServicePrincipal

Get-AzResourceGroup
```

> You should get an output showing all the Azure Resource Groups you have access to

Reviewer: ```suggestion
You should get output showing all the Azure Resource Groups you have access to:
```

> ![Screenshot of Azure Resource Groups](img.jpg)

Reviewer: ```suggestion
![Screenshot of Azure Resource Groups](image.png)
```
